### DISCUSSION TITLE
Why does `navigation.path` not work?

### DISCUSSION THREAD

**Discussion by boundless-forest** [squidfunk/mkdocs-material#7206] [Q&A] [answered]
I am working on a docs project https://github.com/msgport/docs built with this framework and would like to  support Breadcrumbs. I did the config following the tutorial https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#navigation-path, but it doesn't seem to work. I have no idea about the reason.

1. The config: https://github.com/msgport/docs/blob/main/mkdocs.yml#L26
2. The live site https://docs.msgport.xyz/build/tutorial/remix-demo/

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @boundless-forest,
this feature is available to Insiders sponsors only, as indicated by the beating ❤️ emoji.
https://squidfunk.github.io/mkdocs-material/insiders/
https://squidfunk.github.io/mkdocs-material/conventions/#sponsors-only

The linked website was built with the free community version.

**Reply by kamilkrzyskow** [reply]
I forgot to mention that you could add a similar feature using template overrides:
https://squidfunk.github.io/mkdocs-material/customization/#extending-the-theme

The implementation in Insiders is in `base.html`, but I guess you could use `content.html` as well :v:

**Comment by soragXYZ** [comment]
Hello, do we know when this feature will become available for regular folks ?

**Reply by squidfunk** [reply]
Once [the next funding goal](https://squidfunk.github.io/mkdocs-material/insiders/benefits/#18000-lemon-drop) is hit ☺️
